Dockerfile

```diff
@@ -8,7 +8,6 @@ COPY . /go/src/github.com/seatgeek/docker-mirror/
 RUN dep ensure -vendor-only
 RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o build/docker-mirror  .
 
-FROM alpine:latest
-RUN apk --no-cache add ca-certificates
```

Comment:
i don't think we have to install ca-certificates when we're using docker-base. my only evidence of that is that other projects which use docker-base don't install ca-certificates.

Reply:
yeah this is ok, i just built the docker image locally and ran:
```
root@56a97be74b79:/# ls /usr/share/ca-certificates/mozilla/ | wc -l
138
```